consumerBuilder_test.go

```diff
@@ -155,6 +157,19 @@ func (s *ConsumerBuilderTestSuite) TestBuild() {
 		sort.Strings(output)
 		return output
 	}())
+	// make sure retry topic got populated to the read topic list
```

Comment:
nit: can you change this comment to "make sure retry topic with delay gets populated ..."? Seems like you're checking specifically that "retry-topic" is a topic with non-zero `topic.Delay`.